src/modules/keycloak-administration/repository/user-lock.repository.ts

```diff
@@ -62,4 +62,15 @@ export class UserLockRepository {
     public async deleteUserLock(personId: string, lockOccasion: PersonLockOccasion): Promise<void> {
         await this.em.nativeDelete(UserLockEntity, { person: personId, locked_occasion: lockOccasion });
     }
+
+    public async getLocksToUnlock(): Promise<UserLock[]> {
+        const today: Date = new Date();
+
+        const filters: QBFilterQuery<UserLockEntity> = {
+            locked_until: { $lte: today },
```

Comment:
Are we sure about this? The saved lockedUntil in the table userLock is in UTC and not in german time. Here we compare the german time (which is UTC + 1 )  to the UTC time.  

We could maybe use ` const now = DateTime.utc();` instead.

@kristoff-kiefer  @marode-cap What do you think?

Reply:
Pretty sure `DateTime` is not a JS-API. When calling the `Date`-constructor without any arguments, the internal timestamp will be the same as if I used `Date.now()`, which is in UTC.
I think MikroORM only compared the actual timestamp, but I couldn't say for sure

Reply:
@DeFi3298 Can you maybe check what "today" is according to MikroORM? To make sure we are actually comparing UTC with UTC.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date

"When no parameters are provided, the newly-created Date object represents the current date and time as of the time of instantiation. The returned date's timestamp is the same as the number returned by Date.now()"